.travis.yml

```diff
@@ -2,8 +2,8 @@ language: java
 jdk:
   - oraclejdk8
 script:
+  - cd api/
   - ./gradlew check
-  - ./gradlew integrationTest
   - ./gradlew jacocoTestReport
```

Comment:
Could you please add building frontend as well?

- `yarn`

Reply:
Sure.